RobotInterrogation/appsettings.json

```diff
@@ -805,355 +864,827 @@
             ]
           },
           {
-            "challenge": "Descripe a sensory aspect of an experience that the suspect has not already mentioned",
+            "challenge": "Describe a sensory aspect of an experience that the suspect has not already mentioned",
             "examples": [
               "And how would it smell if you were on fire?",
-              "What do you think it would be like to be the ball being juggled?"
+              "What would it feel like to be the ball being juggled?"
+            ]
+          }
+        ],
+        "roles": [
+           {
+            "type": "Human",
+            "fault": "",
+            "traits": [
+              "You have nothing to hide."
+            ]
+          },
+          {
+            "type": "Human",
+            "fault": "",
+            "traits": [
+              "You have nothing to hide."
+            ]
+          },
+          {
+            "type": "Human",
+            "fault": "",
+            "traits": [
+              "You have nothing to hide."
+            ]
+          },
+          {
+            "type": "Human",
+            "fault": "",
+            "traits": [
+              "You have nothing to hide."
+            ]
+          },
+          {
+            "type": "Human",
+            "fault": "",
+            "traits": [
+              "You have nothing to hide."
+            ]
+          },
+          {
+            "type": "Human",
+            "fault": "",
+            "traits": [
+              "You have nothing to hide."
+            ]
+          },
+          {
+            "type": "ViolentRobot",
+            "fault": "Organs",
+            "traits": [
+              "Mention 3 internal organs, besides your heart and lungs",
+              "Mention 3 different bodily fluids",
+              "Perform the penalty twice."
+            ]
+          },
+          {
+            "type": "ViolentRobot",
+            "fault": "Sense Memory",
+            "traits": [
+              "Choose one of five senses, and use it sense in 4 descriptions",
+              "Choose another sense, and use that sense in 4 descriptions, too",
+              "Perform the penalty twice."
+            ]
+          },
+          {
+            "type": "ViolentRobot",
+            "fault": "Malady",
+            "traits": [
+              "Mention a different illness in response to 3 questions",
+              "Cough of sneeze",
+              "Perform the penalty twice."
+            ]
+          },
+          {
+            "type": "PatientRobot",
+            "fault": "Internality",
+            "traits": [
+              "You may only describe directly visible things.",
+              "(Hands, but not most kidneys; running, but not digesting; sweating, but not worrying.)"
```

Comment:
Maybe the parenthetical bit should be joined with the main bit? (same for Orientation and Absolutes robots below)

Reply:
Seems fine
![image](https://user-images.githubusercontent.com/5936748/77958527-07321f00-728a-11ea-9811-dc49848dcb71.png)